packages/core/core/src/requests/AssetRequest.js

```diff
@@ -57,9 +68,7 @@ async function run({input, api, options, farm}: RunInput) {
   // These are used to compute the cache key for assets during transformation.
   request.invalidations = api.getInvalidations().filter(invalidation => {
     // Filter out invalidation node for the input file itself.
-    return (
-      invalidation.type !== 'file' || invalidation.filePath !== input.filePath
-    );
+    return invalidation.type !== 'file' || invalidation.filePath !== realpath;
```

Comment:
Subtle one: when a file was behind a symlink an extra invalidation was created (which needed to be read and hashed during transformation), because we compared the original path instead of the realpath (which is stored in the File node).